scipy/io/_harwell_boeing/hb.py

```diff
@@ -221,7 +221,8 @@ def __init__(self, title, key,
         if key is None:
             key = "|No Key"
         if len(key) > 8:
-            warnings.warn("key is > 8 characters (key is %s)" % key, LineOverflow)
+            warnings.warn("key is > 8 characters (key is %s)" % key,
+                          LineOverflow, stacklevel=2)
```

Comment:
```suggestion
                          LineOverflow, stacklevel=3)
```
Looks like the `HBInfo` class isn't public.